pkg/virt-controller/watch/migration.go

```diff
@@ -704,8 +707,29 @@ func (c *MigrationController) handleTargetPodCreation(key string, migration *vir
 	// migration was accepted into the system, now see if we
 	// should create the target pod
 	if vmi.IsRunning() {
-		if err := c.createPDB(key, vmi, migration); err != nil {
-			return err
+		if migrations.MigrationNeedsProtection(vmi) {
+			pdbs, err := pdbs.PDBsForVMI(vmi, c.pdbInformer)
+			if err != nil {
+				return err
+			}
+			if len(pdbs) < 1 {
+				log.Log.Object(vmi).Errorf("Found no PDBs protecting the vmi")
+				return err
```

Comment:
isn't `err` nil here?

Reply:
Ouch, you're right, I'll fix it, thanks!